shared/teams/team/rows/member-row/index.tsx

```diff
@@ -66,6 +67,8 @@ export const TeamMemberRow = (props: Props) => {
     if (props.status === 'deleted') {
       resetLabel = 'Has deleted their account'
     }
+  } else if (props.needsPUK) {
+    resetLabel = ' • Needs to update Keybase'
```

Comment:
Will this also show for web-only users? If so do we want to broaden this message?

Reply:
Yeah, I dunno, we're planning a larger change to fully distinguish that case. I'd rather not add too much text since the space is small on mobile and figure that the need to install it is arguably implied by needing to "update"